### PR TITLE
Create CHANGELOG

### PR DESCRIPTION
**Pull request by abrauchli**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+ * [`added`]    Support low power mode on SHTC1 (and family)
+
+## [2.0.0] - 2018-05-14
+
+ * First public release
+
+[Unreleased]: https://github.com/Sensirion/embedded-sgp/compare/3.0.0...master
+[3.0.0]: https://github.com/Sensirion/embedded-sgp/compare/2.1.0...3.0.0
+[2.1.0]: https://github.com/Sensirion/embedded-sgp/compare/2.0.0...2.1.0
+[2.0.0]: https://github.com/Sensirion/embedded-sgp/releases/tag/2.0.0
```
Comment (rnestler) [inline CHANGELOG.md]: The links point to the sgp repository (and are thus invalid).

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+ * [`changed`]  Return types are now `int16_t` instead of `int8_t` (in line with
+                other embedded drivers).
+ * [`changed`]  Functions are now dedicated per sensor (e.g. `sht3x_probe()`
+                instead of `sht_probe()`)
+ * [`added`]    New return values for wakeup/sleep failed
+                (`STATUS_WAKEUP_FAILED`, `STATUS_SLEEP_FAILED`)
+ * [`fixed`]    Fix SHTC3 issues when calling `probe()` multiple times
+ * [`changed`]  Move the i2c init call out of `probe()` and into the example
+ * [`changed`]  Rename `git_version.[ch]` to `sht_git_version.[ch]`
+ * [`changed`]  Dedicated sht commands per sensor type
```
Comment (rnestler) [inline CHANGELOG.md]: Isn't this a duplicate of the" Functions are now dedicated per sensor" entry?

File: CHANGELOG.md (hunk 1 above)
Comment (abrauchli) [reply CHANGELOG.md]: darn you copy-paste

File: CHANGELOG.md (hunk 2 above)
Comment (abrauchli) [reply CHANGELOG.md]: yup, I probably thought it's twice as important :roll_eyes:

**Pull request merged**